src/pages/_document.tsx

```diff
@@ -1,36 +1,47 @@
 import NextDocument, { Html, Main, NextScript, Head } from 'next/document'
 import { GTM_ID } from '@/lib/analytics'
+import Script from 'next/script'
 
-export default class Document extends NextDocument {
-  render() {
-    return (
-      <Html lang="en" dir="ltr">
-        <Head>
-          <meta charSet="utf-8" />
-          <link
-            rel="stylesheet"
-            data-entry-name="style.css"
-            href="https://unpkg.com/@department-of-veterans-affairs/formation/dist/formation.min.css"
+const Document = () => {
+  return (
+    <Html lang="en" dir="ltr">
+      <Head>
+        <meta charSet="utf-8" />
+        <link
+          rel="stylesheet"
+          data-entry-name="style.css"
+          href="https://unpkg.com/@department-of-veterans-affairs/formation/dist/formation.min.css"
+        />
+        <link
+          rel="stylesheet"
+          data-entry-name="style.css"
+          href="https://prod-va-gov-assets.s3-us-gov-west-1.amazonaws.com/generated/style.css"
+        />
+        <Script
+          id="staticPages"
+          strategy="afterInteractive"
+          src="https://prod-va-gov-assets.s3-us-gov-west-1.amazonaws.com/generated/static-pages.entry.js"
+        />
+        <Script
```

Comment:
will you swap the order of these two and/or add a comment explaining that the import order here matters, because we need the data added to the window object (or whatever the reason actually is)